curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60f805f813eaf2049bc2ceea.md

```diff
@@ -26,25 +26,33 @@ assert.equal(document.querySelectorAll('label input')?.length, 4);
 You should add the first `input` after the `label` text `Enter Your First Name:`, and include a space after the colon.
 
 ```js
-assert.equal(document.querySelectorAll('label')?.[0]?.innerHTML.trim(), 'Enter Your First Name: <input>');
+const query = /Enter Your First Name:\s*<input>/
```

Comment:
Will this allow for people writing their input tags as `<input/>`?

I thought we generally wanted to use the `querySelectorAll` functionality?

Reply:
For the first question, yes. 

As for the second question, document.querySelectorAll is being used down below. What am I doing is performing a match against the innerHtml to the query I've made in order to handle any additional spacing problems.

Reply:
I deleted the second question - I understood what I read a bit better after I posted the question

Reply:
> Will this allow for people writing their input tags as `<input/>`?

Here's a picture btw. 

![Screenshot](https://imgur.com/3UuU2Iy)